### PR TITLE
Rework the case summary flow.

### PR DESCRIPTION
**Pull request by jaleszek**
Primary Assessor's Case Summary isn't visible to the Lead Assessor
(Lead Assessor should be able to see everything the primary and
secondary are doing).

### CODE REVIEW

File: app/helpers/case_summary_helper.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module CaseSummaryHelper
+  def visible_case_summaries(subject, form_answer)
+    all = [
+      wrap_case_summary(primary_case_summary_assessment, "Primary Case Summary"),
+      wrap_case_summary(lead_case_summary_assessment)
+    ]
+    return all if subject.is_a?(Admin)
+
+    primary = subject.primary?(form_answer)
+    lead    = subject.lead?(form_answer)
```
Comment (matthewford) [inline app/helpers/case_summary_helper.rb]: Unnecessary spacing detected.

**Comment by RuslanHamidullin** [comment]
@jaleszek this one need to rebase ^

**Comment by RuslanHamidullin** [comment]
Looks good :+1: Need to rebase

**Comment by jaleszek** [comment]
@dreamfall can you merge it prob. as Ruslan already has reviewed this ?

**Pull request merged**